src/key2/BrickFloor.java

```diff
@@ -0,0 +1,10 @@
+package key2;
+
+public class BrickFloor implements Floor {
+
+	@Override
+	public String getRepresentation() {
+		return "Finished building wood floor";
```

Comment:
getRepresentation implementation and BrickFloor should not refer to any "building" aspect, that's our builder responsability. Please take a look at  the Single Responsability Principle http://en.wikipedia.org/wiki/SOLID_(object-oriented_design). Keep this in mind for next Topic.